service/service.go

```diff
@@ -80,7 +80,7 @@ func New(config Config) (*Service, error) {
 	{
 		c := k8sclient.ClientsConfig{
 			SchemeBuilder: k8sclient.SchemeBuilder{
-				apiv1alpha3.AddToScheme,
+				capi.AddToScheme,
```

Comment:
Could we even remove the `cluster-api``dependency? I don't think it's used?

Reply:
This makes sense, I didn't check where it's been used. You're right it's not even used 🙈. I dropped it.